data/docs/instrumentation/opentelemetry-react-native.mdx

```diff
@@ -0,0 +1,196 @@
+---
+date: 2025-09-11
+id: opentelemetry-react-native
+tags : [SigNoz Cloud, Self-Host]
+title: React Native OpenTelemetry Instrumentation
+description: Send events from your React Native application to SigNoz
+hide_table_of_contents: true
+---
+
+This document contains instructions on how to set up OpenTelemetry instrumentation in your React Native Applications and view your application traces in SigNoz.
+
+<Admonition type="warning">
+The JS OTel packages are supported for node and web environments. While they work for React Native as well, they are not explicitly supported for that environment, where they might break compatibility with minor version updates or require workarounds. Building JS OTel package support for React Native is an area of active development.
+</Admonition>
+
+**Step 1.** Install OpenTelemetry packages
+
+```bash
+npm install --save @opentelemetry/api@^1.9.0
```

Comment:
I understand the point made regarding compatibility. But please verify if we can have more flexibility regarding this. Test out the latest versions once and see if it still works